controllers/controllers.go

```diff
@@ -63,7 +71,14 @@ func (r *instanceReconciler) ensureInstanceIsUpToDate(instanceInfo *instance.Inf
 			instanceInfo.Node.GetAnnotations()[metadata.VersionAnnotation])
 		return nil
 	}
-
+	// Check if the running version of WMCO is one where in tree storage is deprecated in favor of CSI.
+	csiEnabled, err := r.usesCSI()
+	if err != nil {
+		return err
+	}
+	if csiEnabled {
+		annotationsToApply[csiAnnotation] = "true"
+	}
```

Comment:
Since `usesCSI()` is only needs the `instanceReconciler` platform, could we parameterize the platform and make this a regular helper function? This way we call it a single time from `ensureInstancesAreUpToDate()` instead of once per node (and pass in the csiAnnotation to this func as part of annotationsToApply)